cockroachcloud/migrations-page.md

```diff
@@ -83,23 +78,87 @@ The dump file must be smaller than 4 MB. `INSERT` and `COPY` statements will be
+1. When analysis is complete, review the [**Summary Report**](#summary-report) and edit, add, or remove SQL statements in the [**Statements** list](#statements-list).
+
+<section class="filter-content" markdown="1" data-scope="postgres mysql">
+#### Use Credentials
+
+The Schema Conversion Tool can connect directly to a PostgreSQL or MySQL database to obtain the schema. To add a schema using credentials:
+
+1. In step 2 of the **Add SQL Schema** dialog, click **Use Credential**. Select the credentials to use. If the list is empty, this is because no credentials have been created for the selected database type. You can [add credentials](#add-database-credentials) from the pulldown menu.
+1. Click **Convert** and wait for the schema to be analyzed. In the background, the Schema Conversion Tool runs the [`pg_dump`](https://www.postgresql.org/docs/current/app-pgdump.html) or [`mysqldump`](https://dev.mysql.com/doc/refman/8.0/en/mysqldump.html) utility to obtain the schema.
+
+    A loading screen is displayed. Depending on the size and complexity of the SQL dump, analyzing the schema can require up to several minutes.
+
+1. When analysis is complete, review the [**Summary Report**](#summary-report) and edit, add, or remove SQL statements in the [**Statements** list](#statements-list).
+
+</section>
+
+## Credentials table
+
+If you have [added any database credentials](#add-database-credentials), the following details are displayed when the **Credentials** tab is open:
+
+|      Column     |                                                    Description                                                    |
+|-----------------|-------------------------------------------------------------------------------------------------------------------|
+| Credential Name | The name associated with the access credentials. A `VERIFIED` badge will display if the credentials are verified. |
+| Dialect         | The type of database being accessed.                                                                              |
+| Host / Port     | The host and port used to access the database.                                                                    |
+| Database Name   | The name of the database being accessed.                                                                          |
+| Created At      | The timestamp when the credentials were successfully created.                                                     |
+
+To delete or verify a set of credentials, select the appropriate option in the **Actions** column.
+
+### Add database credentials
+
+{{site.data.alerts.callout_info}}
+Credentials can be added for PostgreSQL and MySQL databases.
+{{site.data.alerts.end}}
+
+1. With the **Credentials** tab open, click the **Add Credential** button.
+1. Provide the following information:
+    - A **Credential Name** to associate with the credentials.
```

Comment:
Do we need to explain how to get this info from the source cluster?

Reply:
Hmm, I think it's implicit that people have to consult the docs for their source db. I'm hesitant to link out since a) the links could go stale, and b) it seems like a fairly basic topic. I'm open to disagreement here though.